File: java-api-core/src/com/kinvey/java/store/PullTaskResponse.java
package com.kinvey.java.store;


import com.kinvey.java.Query;
import com.kinvey.java.model.KinveyReadResponse;

class PullTaskResponse {

    private KinveyReadResponse kinveyReadResponse;
    private Query query;

    PullTaskResponse(KinveyReadResponse kinveyReadResponse, Query query) {
        this.kinveyReadResponse = kinveyReadResponse;
        this.query = query;
    }

    KinveyReadResponse getKinveyReadResponse() {
        return kinveyReadResponse;
    }

    Query getQuery() {
        return query;
    }
}

File: java-api-core/src/com/kinvey/java/store/PullTaskResponse.kt
package com.kinvey.java.store

import com.google.api.client.json.GenericJson
import com.kinvey.java.Query
import com.kinvey.java.model.KinveyReadResponse

data class PullTaskResponse<T>(
    val kinveyReadResponse: KinveyReadResponse<T>?,
    val query: Query?
)
